api-report/sequence.api.md

```diff
@@ -123,10 +123,16 @@ export class IntervalCollection<TInterval extends ISerializableInterval> extends
     [Symbol.iterator](): IntervalCollectionIterator<TInterval>;
     constructor(helpers: IIntervalHelpers<TInterval>, requiresClient: boolean, emitter: IValueOpEmitter, serializedIntervals: ISerializedInterval[]);
     // (undocumented)
+    ackAdd(serializedInterval: ISerializedInterval, local: boolean, op: ISequencedDocumentMessage): TInterval;
```

Comment:
do we need all these methods. could we just have a single ack method, and collaspe the handler into it, rather than separating the two?

Reply:
either way i think we should mark these as @internal

Reply:
marked them as internal. Might do more cleanup here if I end up collapsing more of the defaultMap/intervalCollection boundary, but with current structure of that this seems to make the most sense.